pytrip/volhist.py

```diff
@@ -109,6 +115,13 @@ def volume_histogram(data_cube, voi=None, bins=256):
             vcube = voi.get_voi_cube()
             mask = (vcube.cube == 1000)
 
+        if True not in mask:
+            warnings.warn("Given VOI has no extend and contains no voxels.",
+                          UserWarning)
+            x = np.arange(100, dtype=np.float64)
+            y = np.zeros(100, dtype=np.float64)
+            return x, y
```

Comment:
I think we should return here `None,None` and handle it properly on the other (GUI) side